lineapy/transformer/transform_code.py

```diff
@@ -22,9 +27,30 @@ def transform(
     if it exists.
     """
 
-    transformers = [
-        NodeTransformer(code, location, tracer),
-    ]
+    # create sourcecode object and register source code to db
+    src = SourceCode(id=get_new_id(), code=code, location=location)
+    tracer.db.write_source_code(src)
+
+    # defaults for executor that were set inside node transformer
+    # Set __file__ to the pathname of the file
+    if isinstance(location, Path):
+        tracer.executor.module_file = str(location)
+
+    # initialize the transformer IN ORDER of preference
```

Comment:
clarify transformers will be run in order they are listed here

Reply:
done in next commit.